Initial commit with all files.

jayPare opened the pull request.

jayPare committed:
Initial commit with all files.

jayPare committed:
Adding USA source country support.

jayPare committed:
Add a bunch of source countries.
Move the "country to code" code to a UMD in its own repo.
Removed unused lib in root/lib.

Helpha commented on discogsFees/feeCalculator.js:
```diff
@@ -0,0 +1,1164 @@
+//TODO: Replace $ with actual currency in breakdown.
+//TODO: Fill ddl with JS instead of 3495493429 options in HTML.
+
+const eeaCountries = [
```
Peut-être avoir juste un seul object Json ?

countries = 
{
"EAA":["Croatia", "Bulgaria"],
"UK" :["Ireland"]
}

Helpha commented on discogsFees/feeCalculator.js:
```diff
@@ -0,0 +1,1164 @@
+  switch (item.source) {
+    // See: https://www.paypal.com/ki/webapps/mpp/merchant-fees
+    //AUD
+    case "Kiribati":
```
A date je peux voir qu'il y a vraiment beaucoup de liste de pays différentes, je ferais peut-être genre une liste d'object pays qui aurait toutes les propriétés possible dedans, feck après ça l'info est toute centralisée à la même place au lieu de redéfinir plusieur fois les shits.

jayPare committed:
Add currency drop down list.
Some HTML modification.
Add several countries.

jayPare committed:
Fix info for Discogs Fees Calculator.

jayPare committed:
Fixing index title.

jayPare committed:
Add support for the following countries:
  - Australia
  - China
  - Hong Kong
  - Israel
  - Japan
  - Mexico
  - Russia
  - South Korea
  - Switzerland
  - Taiwan
  - Thailand
  - United Kingdom
  - Guernsey
  - Isle of Man
  - Jersey

Add GST support for Australia and New Zealand.
Fill country drop-down lists programmatically.
Validate CSV object before using it.
Rename functions to better names.
Updating title.
UI tweaks.
Info and disclamer banners update.

jayPare committed:
UI tweaks.

jayPare committed:
Fixing some typos.

Helpha reviewed (commented):
Stu ça
